crates/wallet/src/wallet/mod.rs

```diff
@@ -1136,15 +1137,33 @@ impl Wallet {
     }
 
     /// Iterate over the transactions in the wallet.
-    pub fn transactions(
-        &self,
-    ) -> impl Iterator<Item = CanonicalTx<'_, Arc<Transaction>, ConfirmationTimeHeightAnchor>> + '_
-    {
+    pub fn transactions(&self) -> impl Iterator<Item = WalletTx> + '_ {
         self.indexed_graph
             .graph()
             .list_chain_txs(&self.chain, self.chain.tip().block_id())
     }
 
+    /// Array of transactions in the wallet sorted with a comparator function.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// # use bdk_wallet::Wallet;
+    /// # use bdk_wallet::wallet::WalletTx;
+    /// # let changeset = Default::default();
+    /// # let mut wallet = Wallet::load_from_changeset(changeset)?;
+    /// // Transactions by chain position: first unconfirmed then descending by confirmed height.
+    /// let sorted_txs:Vec<WalletTx> = wallet.transactions_sort_by(|tx1, tx2| tx2.chain_position.cmp(&tx1.chain_position));
+    /// # Ok::<(), anyhow::Error>(())
```

Comment:
We are missing a closing ```
```suggestion
    /// # Ok::<(), anyhow::Error>(())
    /// ```
```

Reply:
oops! fixed.